src/main/java/core/basesyntax/FlipBit.java

```diff
@@ -8,7 +8,26 @@
      * Данная задача актуальна, например, при работе с битовыми полями.
      * Договоримся, что биты нумеруются от младшего (индекс 1) к старшему (индекс 32).</p>
      */
-    public int flipBit(int value, int bitIndex) {
-        return 0;
+    public static int flipBit(int value, int bitIndex) {
```

Comment:
you had to use bitwise operators for this task. didn't you see the [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_02.html)?